php7.3-phpunit7/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM php:7.3-cli
```

Comment:
I don't understand why we need to have 2 copies of the dockerfile, since you've moved to a model where it's building out of the top-level directory instead of these subdirs.

Reply:
I really did not know enough and was just guessing that having separate directories for each version was unnecessary.  So I was testing if I could remove the directories.  And it look like I can.

Since this repo is now only being used by the tech center, I suspect there is no reason to keep those extra directories and their content.

Reply:
I removed the exteraneous files and directories and rebuilt it.